node/core/pvf/src/execute/worker_intf.rs

```diff
@@ -73,18 +75,23 @@ pub enum Outcome {
 	InvalidCandidate { err: String, idle_worker: IdleWorker },
 	/// An internal error happened during the validation. Such an error is most likely related to
 	/// some transient glitch.
-	InternalError { err: String, idle_worker: IdleWorker },
+	///
+	/// Should only ever be used for errors independent of the candidate. Therefore it may be a problem with the worker,
+	/// so we terminate it.
+	InternalError { err: String },
```

Comment:
And independent of the PVF. It has to be guaranteed to not be triggered by either a PVF or a candidate.